portal/src/components/Materials/MaterialInfo/MaterialInfo.component.js

```diff
@@ -123,5 +129,15 @@ export default {
       return false;
     }
   },
-  watch: {}
+  watch: {
+    /**
+     * If the material changes, it is checked if the material has been rated
+     */
+    material: function () {
+      const { material } = this;
+      if (material) {
```

Comment:
Als ik nu een lege Array of Object als material meegeef dan triggert dit. Dat is waarschijnlijk niet wat je bedoeld hebt en kan tot gekke bugs gaan leiden. Je kunt beter isNil importeren van lodash. Dan check je op null of undefined.